### PR TITLE
Homepage enhancements

### PR DESCRIPTION
**Pull request by hblitza**
Livepreview: https://hblitza.github.io/react-geo/

### Description:

Added Bootstrap-Layout to the gh-pages landing page.

### CODE REVIEW

**Comment by KaiVolland** [comment]
Plz don't use a terrestris logo but the `react-geo` [logo](https://github.com/terrestris/react-geo/blob/master/assets/logo.svg).

**Comment by marcjansen** [comment]
I think we can also list terrestris as main contributor. that doesn't hurt. it might be we need to move this to the master branch as well. we'll sort this out tomorrow

**Comment by hblitza** [comment]
I deleted the terrestris logo in the footer. I think the terrestris copyright text refers to the website, not to react-geo. Should it be deleted as well?

File: examples/examples.html [hunk 1]
```diff
@@ -1,15 +1,133 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (ahennr) [inline examples/examples.html]: Isn't this file autogenerated by `Metalsmith`? Changes here would be overwritten each time the examples are build, or am I  wrong? Ping @KaiVolland

Comment (marcjansen) [reply examples/examples.html]: Probably right. We might need to take the changes and apply them to files in the master branch. @hblitza please ask @KaiVolland for directions.

**Comment by marcjansen** [comment]
Please talk to @KaiVolland and @ahennr for input on the frontpage. The contents of the README might be a good starting point. Thx, good work so far :clap:

**Comment by dnlkoch** [comment]
Thanks for this!

**Pull request merged**